README.md

```diff
@@ -2,24 +2,220 @@
+
+Note that attributes can be proposed for elevation separate from the APIs that use them. Promotion of one or more attributes independent of any API may, for example, occur when a "Stable" API has expressed a desire to utilize them. In such cases, the proposal should clearly identify the "Stable" APIs making the request and provide some justification for it.
+
+When ready for consideration, the author(s) shall label the PR with the "Elevate" tag, thereby requesting that the matter be placed on the agenda of the next quarterly meeting (subject to the one month deadline). As with the process for Provisional acceptance, the "**Straw Poll**" mechanism is utilized to provide an initial screening of the proposal. Passing of the straw poll _does not_ confer elevation to Stable status, but serves instead as a means for quickly pushing back proposals that lack significant support. This is intended to both protect the time of ASC members and author(s), and to help guide author(s) wishing to continue pursuit of the proposal.
+
+Upon successfully completing the straw poll, the Co-Chairs shall label the PR as "**Eligible**" to indicate that the proposal is eligible for formal consideration by the ASC. Given the significant commitment conferred by the PMIx community to "Stable" elements, formal approval of elevation requires that the ASC conduct a full presentation on the proposal at two consecutive quarterly meetings. At the conclusion of the first meeting, a formal vote of the ASC shall be held (per the above voting rules) to determine if the proposal merits further consideration. The Co-Chairs shall set a GitHub label indicating the result as either "**First Pass**" or "**Pushed Back**".
+
+At the conclusion of the second meeting where the proposal is up for consideration, another formal vote of the ASC shall be held (per the above voting rules) to determine final disposition of the proposal. The Co-Chairs shall subsequently set a GitHub label indicating the result as either "**Stable**" (indicating that the proposed elements have been elevated to that class) or "**Pushed Back**".
+
+Proposals that are pushed back at any point can be modified and resubmitted by the author(s). However, the updated PR must pass through the complete elevation process from the beginning to ensure full consideration of the revised request. A new PR for this purpose _must not_ be submitted for the identical elements in order to retain full information regarding prior concerns and suggestions.
```

Comment:
I think that this part is missing what the release manager should do once elevation is accepted.

Reply:
Added